tests/Integration/inc/classes/subscriber/third-party/plugins/Smush_Subscriber/maybeDeactivateRocketLazyload.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\classes\third_party\plugins\Smush_Subscriber;
+
+use Brain\Monkey\Functions;
+
+/**
+ * @covers \WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber::maybe_deactivate_rocket_lazyload
+ * @group ThirdParty
+ * @group Smush
+ * @group WithSmush
+ */
+class Test_MaybeDeactivateRocketLazyload extends SmushSubscriberTestCase {
+	private $option_hook_prefix = 'pre_get_rocket_option_';
+	private $rocket_settings;
+
+	public function setUp() {
+		parent::setUp();
+
+		remove_all_filters( $this->option_hook_prefix . 'lazyload' );
+		remove_all_filters( $this->option_hook_prefix . 'lazyload_iframes' );
+		remove_all_filters( 'update_option_wp_rocket_settings' );
```

Comment:
Need to "remember" what is registered to each and then restore at `tearDown()`. Else, it could impact other integration tests.